chart/app-readme.md

```diff
@@ -18,14 +18,16 @@ This chart installs the following components:
-| CIS    | k3s                     | [k3s-cis-1.8-hardened](https://github.com/rancher/security-scan/tree/master/package/cfg/k3s-cis-1.8-hardened)      | k3s-v1.26+          |
-| CIS    | eks                     | eks-1.2.0                                                                                                          | eks                 |
-| CIS    | aks                     | aks-1.0                                                                                                            | aks                 |
-| CIS    | gke                     | gke-1.2.0                                                                                                          | gke                 |
-| CIS    | gke                     | gke-1.6.0                                                                                                          | gke-1.29+            |
+| CIS    | any                     | cis-1.9                                                             | v1.27+              |
+| CIS    | any                     | cis-1.8                                                             | v1.26               |
+| CIS    | rke                     | [rke-cis-1.8-permissive](https://github.com/rancher/security-scan/tree/main/package/cfg/rke-cis-1.8-permissive)   | rke1-v1.26+         |
+| CIS    | rke                     | [rke-cis-1.8-hardened](https://github.com/rancher/security-scan/tree/main/package/cfg/rke-cis-1.8-hardened)                   | rke1-v1.26+         |
+| CIS    | rke2                    | [rke2-cis-1.8-permissive](https://github.com/rancher/security-scan/tree/main/package/cfg/rke2-cis-1.8-permissive)  | rke2-v1.26+         |
+| CIS    | rke2                    | [rke2-cis-1.8-hardened](https://github.com/rancher/security-scan/tree/main/package/cfg/rke2-cis-1.8-hardened)                  | rke2-v1.26+         |
+| CIS    | k3s                     | [k3s-cis-1.9](https://github.com/rancher/security-scan/tree/main/package/cfg/k3s-cis-1.9)                                      | k3s-v1.27+          |
+| CIS    | k3s                     | [k3s-cis-1.8-permissive](https://github.com/rancher/security-scan/tree/main/package/cfg/k3s-cis-1.8-permissive)   | k3s-v1.26           |
+| CIS    | k3s                     | [k3s-cis-1.8-hardened](https://github.com/rancher/security-scan/tree/main/package/cfg/k3s-cis-1.8-hardened)                   | k3s-v1.26           |
+| CIS    | eks                     | eks-1.2.0                                                             | eks                 |
+| CIS    | aks                     | aks-1.0                                                             | aks                 |
+| CIS    | gke                     | gke-1.2.0                                                             | gke                 |
+| CIS    | gke                     | gke-1.6.0                                                             | gke-1.29+            |
```

Comment:
Could you adapt, see this [comment](https://github.com/rancher/cis-operator/pull/520#discussion_r1867808505).

Reply:
done